src/edges_cal/alanmode.py

```diff
@@ -363,6 +364,15 @@ def _average_spectra(
     return spfreq, spectra
 
 
+def get_s11date(datadir, year, day):
+    """Return the name of the s11 date: 2022_318_14 for an input (2022,316)."""
+    file_name = str(
+        get_s1p_files(root_dir=datadir, year=year, day=day, load="open")["input"]
+    ).split("/")[-1]
```

Comment:
Instead of casting the `Path` to a string, use the `Path` methods, e.g. `get_s1p_files(....)['input'].name`